projects/ngrx.io/content/guide/eslint-plugin.md

```diff
@@ -0,0 +1,79 @@
+# ESLint plugin for NgRx
+
+## What is it?
+
+Use [ESLint](https://eslint.org/) to follow the best practices and to avoid common pitfalls in your application.
+
+The [NgRx ESLint Plugin](https://github.com/timdeschryver/eslint-plugin-ngrx) is no different and promotes the key concepts to create a maintainable project. It consists of @ngrx/store, @ngrx/effects, and @ngrx/component-store rules and a handful of preconfigured configurations.
+
+For more information, see the [documentation page](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/README.md).
+
+The plugin comes with a number of rules that help address most popular NgRx malpractices. The rules are configurable so that every developer can choose which ones they necessarily want to follow, and which ones should givbe a linting error or warning, and so on. A detailed documentation of all the rules with examples can be found [here](https://github.com/timdeschryver/eslint-plugin-ngrx/tree/main/docs/rules). Some rules also allow automatic fixes with `ng lint --fix`.
```

Comment:
```suggestion
The plugin comes with a number of rules that help address most popular NgRx malpractices. The rules are configurable so that you can choose the ones you want to follow, and which rules should give a linting error or warning. 

A detailed documentation of all the rules with examples can be found on the [documentation page](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/README.md#rules). 

Some rules also allow automatic fixes with `ng lint --fix`.
```

Reply:
Yeah, using "you" instead of "developer" sounds more friendly, I like it